src/components/Formik/FormikRadioGroup/FormikRadioGroup.test.jsx

```diff
@@ -0,0 +1,185 @@
+import React from 'react';
+import {
+  render,
+  fireEvent,
+  screen,
+  waitFor,
+} from '@testing-library/react';
+import selectEvent from 'react-select-event';
+import { Formik, Form, Field } from 'formik';
+import FormikSelectInput from './FormikSelectInput';
```

Comment:
Wrong component

Reply:
@nathanyoung Yeah, I hadn't started seriously adding tests at that point. Tests will be up for review tomorrow.